config/software/libiconv.rb

```diff
@@ -21,7 +21,6 @@
 default_version "1.14"
 
 dependency "patch" if solaris2?
-dependency "mingw-runtime" if windows?
```

Comment:
Shouldn't this be a bit more careful?  i.e.  If we're installing from devkit, this software definition should do nothing and have a dependency on devkit.  If we aren't, then it should maintain a dependency on mingw-runtime.  This is the software definition needed to satisfy the libiconv requirement.

I guess if it isn't included anywhere on windows...  we might not care?